data_explorer/requirements.txt

```diff
@@ -3,4 +3,6 @@ fondant==0.1.2
 streamlit-aggrid==0.3.4
 pyarrow>=7.0
 matplotlib==3.7.1
-plotly==5.15.0
\ No newline at end of file
+plotly==5.15.0
+fsspec==2023.4.0
+gcsfs==2023.4.0
```

Comment:
if you're not planning on using fondant here maybe you need to install the specific cloud dependencies, see: https://github.com/orgs/ml6team/projects/1?pane=issue&itemId=30868442

Reply:
Good point, I'll add fondant[azure, aws, gcp]

Reply:
I added s3fs and adlfs